paasta_tools/setup_kubernetes_job.py

```diff
@@ -113,8 +115,28 @@ def main() -> None:
         logging.getLogger("kazoo").setLevel(logging.WARN)
         logging.basicConfig(level=logging.INFO)
 
+    # emit deploy events for updated jobs
     deploy_metrics = metrics_lib.get_metrics_interface("paasta")
 
+    # emit timing metrics for s_k_j
+    cluster = args.cluster or load_system_paasta_config().get_cluster()
+    system_metrics = metrics_lib.get_metrics_interface("paasta.setup_kubernetes_job")
```

Comment:
If we start emitting this for more cronjobs, perhaps a more generic prefix is better here, then include the name of the script as a dimension or in the name of the timer below?